Resources/Locale/en-US/_Starlight/species/shadekin.ftl

```diff
@@ -1,14 +1,41 @@
 alerts-shadekin-name = Light Exposure
 alerts-shadekin-desc = How much light is around you.
 
-shadekin-alert-0 = [color=green]Light Exposure: Darkness...[/color]
-                    The darkness is nice... its like home... i feel my wounds healing...
-shadekin-alert-1 = [color=green]Light Exposure: Low[/color]
-shadekin-alert-2 = [color=green]Light Exposure: Annoying[/color]
-                    The light is annoying in this place... i feel my wounds will not heal properly...
-shadekin-alert-3 = [color=green]Light Exposure: High[/color]
-                    too many lights... too many! i feel exausted...
-shadekin-alert-4 = [color=green]Light Exposure:[/color] [color=red]EXTREME[/color]
-                    LIGHTS... I NEED DARKNESS! its burns!
-
-phase-fail-generic = I can't phase!
\ No newline at end of file
+alerts-shadekin-energy-name = Energy
+alerts-shadekin-energy-desc = Your energy level.
+alerts-shadekin-noportal-name = Missing Portal
+alerts-shadekin-noportal-desc = [color=red]You have no portal, you will be unable to rejuvenate![/color]
+alerts-shadekin-rejuvenate-name = Rejuvenating
+alerts-shadekin-rejuvenate-desc = You are currently rejuvenating, you will be able to leave the dark until your at full energy.
```

Comment:
"you will ... until" sounds a bit weird. did you mean "you wont be able to leave the dark until you are at full energy."